wellregistry/registry/admin/bulk_upload.py

```diff
@@ -145,17 +151,23 @@ def post(self, request):
             next(data_stream)
             monitoring_locations = []
             error_messages = []
+            warning_messages = []
             row_index = 1
             for row in csv.reader(data_stream):
                 row_index += 1
                 try:
-                    monitoring_locations.append(_get_monitoring_location(row, request.user))
+                    monitoring_locations.append(_get_monitoring_location(row_index,row, request.user, warning_messages))
```

Comment:
nit pick: space between row_index,row (not me PEP8)

Reply:
fixed

Reply:
fxied